existing.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require 'functions.php';
+
+$db = createDB();
+
+$stmt = $db->query("SELECT `id`, `title`, `image`, `paragraph` FROM `projects`;");
+$projects_array = $stmt->fetchAll();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit existing project</title>
+</head>
+    <body>
+        <form action="upload_projects.php" method="GET">
+            Select Project:
+            <select name="id">
+                <?php
+                foreach ($projects_array as $project) { ?>
+                    <option value="<?php echo $project['id']?>"><?php echo $project['title']; ?></option>
+                <?php } ?>
+            </select>
+            <input type="submit" value="Submit">
+        </form>
+        <form action="delete_projects.php" method="GET">
+            Delete Project:
+            <select name="id">
+                <?php
+                foreach ($projects_array as $project) { ?>
```

Comment:
the select tag and the foreach are identical in this page. put it in a function and call it twice